lib/util/lib/timer.c

```diff
@@ -164,7 +162,7 @@ add_timer(struct timer_list *timer)
         list_del_init(&timer->entry);
     }
 
-    list_for_each_entry(t, &timer_list, entry) {
+    list_for_each_entry (t, &timer_list, entry) {
```

Comment:
Is there no way to prevent this inconsistency/stupidity?

Reply:
This is how it formats `foreach` macros